code/modules/mob/living/living.dm

```diff
@@ -175,7 +175,8 @@
 	var/current_dir
 	if(isliving(AM))
 		current_dir = AM.dir
-	degstepprojectile(AM, t, step_size)
+	var/pd = dir2angle(t)
```

Comment:
So to get here you:
got the degree
turned the degree into a dir
turned the dir into a degree

This seems wasteful, why are you doing it like that?

Reply:
t is the direction, with pixels taken into account and pd is the degree this was wrong previously because it just found the degree and applied it to everything above but move_crush and those procs are expecting a direction

Reply:
this is necessary because get_dir doesn't work on the same tile

Reply:
wait I can clean this up